### PR TITLE
Add docker files, use Ruby 2.3.1, close #126

### LINKED ISSUE #126: Consider adding Docker for DB
Creating dbs and roles manually makes no one happy.

### PR DESCRIPTION
**Pull request by klaseskilson**
Title has it all!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,22 @@ rake db:migrate
...
+docker-compose build
+# start machine
+docker-compose up
+# create database
+docker-compose run web rake db:create
+```
+
+Kör sedan dina kommandon mot Rails genom att skriva `docker-compose run web ` innan kommandona.
+
 ### PostgreSQL setup
```
Comment (tistatos) [inline README.md]: What should we consider standard DB solution?
This part should reflect the changes made in the database config file, by that I mean username and password as well as host?

**Comment by klaseskilson** [comment]
This isn't really compatible with the `letter_opener` gem. I suggest we remove it since the emails are printed in the console. What do you think?

File: README.md (hunk 1 above)
Comment (klaseskilson) [inline README.md]: Good question. I suggest we move towards Docker instead of local setup. Will update PR accordingly.

**Comment by klaseskilson** [comment]
What do you say? Can I get an 🆗  from someone?

**Comment by tistatos** [comment]
LGTM :ship: :it:

**Pull request merged**